src/Controllers/LoginController.php

```diff
@@ -57,7 +57,7 @@ public function loginAction()
      */
     public function logoutAction()
     {
-        $user = auth()->user();
+        auth()->user();
```

Comment:
I think this line can be removed as it is not being used for anything

Reply:
Removed.